packages/google_sign_in/ios/Classes/GoogleSignInPlugin.m

```diff
@@ -120,8 +120,9 @@ - (void)signIn:(GIDSignIn *)signIn
     didSignInForUser:(GIDGoogleUser *)user
            withError:(NSError *)error {
   if (error != nil) {
-    if (error.code == -4) {
-      // Occurs when silent sign-in is not possible, return an empty user in this case
+    if (error.code == -4 || error.code == -5) {
```

Comment:
Are there no constants defined for these error codes? If not, can we define some to make this more readable?